publication-commons/src/main/java/no/unit/nva/publication/service/impl/DynamoDBPublicationService.java

```diff
@@ -305,6 +308,25 @@ private boolean isPublished(Publication publication) {
         return PublicationStatus.PUBLISHED.equals(publication.getStatus());
     }
 
+    @Override
+    public void deletePublication(UUID identifier) throws ApiGatewayException {
```

Comment:
For me it does not build due to "GodClass". Add a PMD exclusion rule for now and a TODO?